.github/workflows/release.yml

```diff
@@ -8,17 +8,18 @@ on:
 jobs:
   build:
     name: Build Polkadot charm
-    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v26.0.1
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v32.0.0
     with:
-      charmcraft-snap-revision: 5780
+      charmcraft-snap-revision: 6672
+      cache: false
 
   release:
     name: Release charm
     needs:
       - build
-    uses: canonical/data-platform-workflows/.github/workflows/release_charm.yaml@v26.0.1
+    uses: canonical/data-platform-workflows/.github/workflows/release_charm_edge.yaml@v32.0.0
     with:
-      channel: latest/edge
+      track: latest
```

Comment:
They have separate files for the different channels now, and thus `edge` does not need to be specified as a parameter anymore.